Flaky Test:  test_try_premium_at_start_new_account_can_pull_data

## Problem Definition

Test fails flakily on Travis:

```
rotkehlchen_instance = <rotkehlchen.rotkehlchen.Rotkehlchen object at 0x7fef7e3b4208>
username = 'testuser', db_password = '123'
    def test_try_premium_at_start_new_account_can_pull_data(
            rotkehlchen_instance,
            username,
            db_password,
    ):
        # Set the can_sync setting to true
        rotkehlchen_instance.data.db.update_premium_sync(True)
        our_last_write_ts = rotkehlchen_instance.data.db.get_last_write_ts()
        assert rotkehlchen_instance.data.db.get_main_currency() == A_USD
        _, our_hash = rotkehlchen_instance.data.compress_and_encrypt_db(db_password)
    
        # remote hash should be different
        remote_hash = 'a' + our_hash[1:]
        remote_data = REMOTE_DATA
        api_key, api_secret, patched_premium, patched_get = create_patched_premium_with_keypair(
            patch_get=True,
            metadata_last_modify_ts=our_last_write_ts + 10,  # Remote DB is newer
            metadata_data_hash=remote_hash,
            saved_data=remote_data,
        )
    
        with patched_premium, patched_get:
            rotkehlchen_instance.try_premium_at_start(
                api_key=api_key,
                api_secret=api_secret,
                username=username,
                create_new=True,
                sync_approval='yes',
            )
    
        # At this point pulling data from rotkehlchen server should have worked
        # and our database should have been replaced. The new data have different
        # main currency
        assert rotkehlchen_instance.data.db.get_main_currency() == A_GBP
        # Also check a copy of our old DB is kept around.
        directory = os.path.join(rotkehlchen_instance.data.data_directory, username)
        files = list(os.path.join(directory, f) for f in os.listdir(directory))
        assert len(files) == 2
>       assert 'rotkehlchen.db' in files[0]
E       AssertionError: assert 'rotkehlchen.db' in '/tmp/pytest-of-travis/pytest-0/data36/testuser/rotkehlchen_db_17_06_2019_22_40_22.backup'
rotkehlchen/tests/test_premium.py:251: AssertionError
```

--- a/rotkehlchen/tests/test_premium.py
+++ b/rotkehlchen/tests/test_premium.py
@@ -248,8 +248,9 @@ def test_try_premium_at_start_new_account_can_pull_data(
     directory = os.path.join(rotkehlchen_instance.data.data_directory, username)
     files = list(os.path.join(directory, f) for f in os.listdir(directory))
     assert len(files) == 2
-    assert 'rotkehlchen.db' in files[0]
-    assert 'backup' in files[1]
+    # The order of the files is not guaranteed
+    assert 'rotkehlchen.db' in files[0] or 'rotkehlchen.db' in files[1]
+    assert 'backup' in files[0] or 'backup' in files[1]
 
 
 def test_try_premium_at_start_new_account_same_hash(

